.rubocop.yml

```diff
@@ -16,6 +16,9 @@ Lint/Debugger:
   Exclude:
   - lib/cc/cli/console.rb
 
+Lint/AssignmentInCondition:
```

Comment:
Why disable this? [Our styleguide](https://github.com/codeclimate/styleguide/blob/master/rubocop.yml) does not